src/c++/perf_analyzer/docs/benchmarking.md

```diff
@@ -0,0 +1,217 @@
+<!--
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Nit: `#` is redundant here, `<!--...-->` already enables a multiline comment.

Reply:
removed